app/src/main/java/grinnell/appdev/edu/lyles/AsyncRetrieval.java

```diff
@@ -0,0 +1,69 @@
+package grinnell.appdev.edu.lyles;
+
+import android.os.AsyncTask;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+/**
+ * Created by Shelby on 2/3/2017.
+ */
+
+public class AsyncRetrieval extends AsyncTask<Void, Void, String> {
+
+    private final String url;
```

Comment:
I like that you put an access modifier here. Good job keeping this `private`. Although, you should also make the member variable follow the convention we agreed on here: https://github.com/GrinnellAppDev/android-guidelines/blob/master/project_and_code_guidelines.md.

This variable name should read : `private final String mURL`;